db/migrate/20220829084501_create_redis_notifications.rb

```diff
@@ -5,6 +5,11 @@ def change
       t.references :subject, polymorphic: true
       t.datetime :published_at
       t.bigint :current_user_id
+      t.bigint :project_id
+      t.bigint :issue_id
+      t.bigint :user_id
+      t.bigint :member_id
+      t.bigint :role_id
```

Comment:
Since this plugin is already published:
If we add the change to an existing migration we force current plugin users to down run the current migration. Sure, there probably won't be many 😄 but I think it's a good practice to add a new migration here.